src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/meta/ConstantPool.java

```diff
@@ -170,6 +170,19 @@ interface BootstrapMethodInvocation {
         List<JavaConstant> getStaticArguments();
     }
 
+    /**
+     * Gets the constant pool index of a static argument of a bootstrap specifier. Used when the list
+     * of static arguments in the {@link BootstrapMethodInvocation} is an {@code int[]}. The list has
+     * two elements. The first one is the number of arguments and the second one is the {@code cpi}.
+     *
+     * @param cpi the index of a bootstrap specifier in the constant pool
```

Comment:
> the index of a bootstrap specifier in the constant pool

What's a bootstrap specifier? That is not mentioned anywhere else so you either have to explain it or use existing terminology.